packages/@uppy/image-editor/src/Editor.jsx

```diff
@@ -140,11 +140,7 @@ export default class Editor extends Component {
           className="uppy-u-reset uppy-c-btn"
           onClick={() => {
             this.cropper.reset()
-            if (opts.cropperOptions.initialAspectRatio) {
-              this.cropper.setAspectRatio(opts.cropperOptions.initialAspectRatio);
-            } else {
-              this.cropper.setAspectRatio(0);
-            }
+            this.cropper.setAspectRatio(opts.cropperOptions.initialAspectRatio);
```

Comment:
Does it go through the linter fine? I think it should compain about `;`

Reply:
I'm sorry, it was my oversight.

Reply:
Can I pass the review now?